packages/address/src/AddressFormatter.ts

```diff
@@ -10,11 +10,19 @@ const DEFAULT_FORM_LAYOUT =
 const DEFAULT_SHOW_LAYOUT =
   '{lastName} {firstName}_{company}_{address1} {address2}_{city} {province} {zip}_{country}_{phone}';
 
-const ORDERED_COUNTRIES_CACHE: {
+let countriesCache: {
   [locale: string]: Country[];
 } = {};
 
 export default class AddressFormatter {
+  /**
+   * Useful in tests or any situation where the cache has undesirable
+   * side-effects.
+   */
+  static resetCache() {
+    countriesCache = {};
```

Comment:
Rather than recreate the whole object you could empty it out:

```
for (var key in ORDERED_COUNTRIES_CACHE) {
  if (ORDERED_COUNTRIES_CACHE.hasOwnProperty(key)) {
    delete ORDERED_COUNTRIES_CACHE[key];
  }
}
```

That would allow it to continue to be a `const` and you wouldn't need to rename the variable

Reply:
Converting it to be a Map sounds like a great idea. I like it.